src/schema.ts

```diff
@@ -0,0 +1,56 @@
+//GraphQl Schema
+//para colocar input (input NomeInput)
+export const typeDefs = `#graphql
+
+  type Query{
+    hello: String
+  }
+
+  #input obrigatório !
+  type Mutation{
+    createUser(input: CreateUserInput!): User
+    login(input: LoginInput!): Login
+}
+  
+  input CreateUserInput{
+    name: String!
+    email: String!
+    password: String!
+    birthDate: String!
+  }
+
+  type Login{
+    token: String!
+    user: User!
+  }
+
+   type User {
+    id: ID
+    name: String
+    email: String
+    birthDate: String
+  }
```

Comment:
esses campos são obrigatórios em um user, certo?
```suggestion
   type User {
    id: ID!
    name: String!
    email: String!
    birthDate: String!
  }
```